problem-3/server/public/index.php

```diff
@@ -0,0 +1,27 @@
+<? php
+
+require '../vendor/autoload.php';  
+ 
+use Teamleader/CodingTest/Logger;
+$logger = new Logger();
+
+// this is an example php file, so that the the project structure is clear to candidates
+
+// you may assume this code connects to some database with the following credentials
+
+$database_host = "127.0.0.1";
```

Comment:
We could expand the test to pass these configuration values to environment values.